tests/Contrib/Unit/ZipkinToNewrelicExporterTest.php

```diff
@@ -4,6 +4,8 @@
 
 namespace OpenTelemetry\Tests\Contrib\Unit;
 
+use GuzzleHttp\Client;
```

Comment:
I'd make at least one test suite use the alternate client and/or factory implementation since they're installed.

Reply:
Looks good @dkarlovi, that would help users see the usage with other factory implementation. Would do that in an incoming example